components/coding/profileV2/BadgesDisplayedComponent.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from "@apollo/client";
+import { format } from "date-fns";
+import React, { useState } from "react";
+import { FETCH_TOTAL_USER_BADGES_COUNT } from "../../../graphql/fetchTotalUserBadgesCount";
```

Comment:
let's remove all our unused imports - we aren't using any gql in this component, just passing down props into it so it can render them

vithushan and I were talking about introducing some automated checks to make sure we don't have console.logs or unused imports which would make these things easier to spot

Reply:
that's a great idea! I want to learn how to write those github actions as well:)